README.md

```diff
@@ -1,24 +1,50 @@
-MusicBrainz Picard
-==================
+# MusicBrainz Picard
+
 ![Github Actions Status](https://github.com/metabrainz/picard/workflows/Run%20tests/badge.svg)
 [![Codacy Grade](https://img.shields.io/codacy/grade/4dcabf0a13ed4b27b3a381ce9ba96ecc/master.svg?style=flat-square&label=Codacy)](https://app.codacy.com/gh/metabrainz/picard)
 
-[MusicBrainz Picard](http://picard.musicbrainz.org) is a cross-platform (Linux, macOS, Windows) application written in Python and is the official [MusicBrainz](http://musicbrainz.org) tagger.
+[MusicBrainz Picard](http://picard.musicbrainz.org) is a cross-platform (Linux, macOS, Windows) audio tagging application. It is the official [MusicBrainz](http://musicbrainz.org) tagger.
 
 Picard supports the majority of audio file formats, is capable of using audio fingerprints ([AcoustIDs](http://musicbrainz.org/doc/AcoustID)), performing CD lookups and [disc ID](http://musicbrainz.org/doc/Disc_ID) submissions, and it has excellent Unicode support. Additionally, there are several plugins available that extend Picard's features.
 
-When tagging files, Picard uses an album-oriented approach. This approach allows it to utilize the MusicBrainz data as effectively as possible and correctly tag your music. For more information, [see the illustrated quick start guide to tagging](https://picard.musicbrainz.org/quick-start/).
+When tagging files, Picard uses an album-oriented approach. This approach allows it to utilize the MusicBrainz data as effectively as possible and correctly tag your music. For more information, [see the illustrated quick start guide to tagging](https://picard.musicbrainz.org/quick-start/) and the [documentation](https://picard-docs.musicbrainz.org/).
 
-Picard is named after Captain Jean-Luc Picard from the TV series Star Trek: The Next Generation.
 
-Binary downloads are available [here](http://picard.musicbrainz.org/downloads/).
+## Features
 
-Support and issue reporting
----------------------------
+- **Multiple formats:** Picard supports all popular music formats,
+  including MP3, FLAC, OGG, M4A, WMA, WAV, and more.
+- **AcoustID:** Picard uses [AcoustID](http://musicbrainz.org/doc/AcoustID)
+  audio fingerprints, allowing files to be identified by the actual
+  music, even if they have no metadata.
+- **Comprehensive database:** Picard uses the open and
+  community-maintained [MusicBrainz database](http://musicbrainz.org)
+  to provide accurate information about millions of music releases.
+- **CD lookups:** Picard can lookup entire music CDs with a click.
+- **Plugin support:** If you need a particular feature, you can choose
+  from a selection of available plugins or write your own.
+- **Scripting:** A flexible but easy to learn scripting language allows
```

Comment:
Perhaps "A flexible and powerful, but easy to learn, scripting language..."?

Reply:
Yes, I like this. I updated this sentence